example-projects/poem-example/src/main.rs

```diff
@@ -0,0 +1,102 @@
+use cloudevents::Event;
+use poem::listener::TcpListener;
+use poem::middleware::Tracing;
+use poem::{get, handler, Endpoint, EndpointExt, Response, Route, Server};
+
+#[handler]
+async fn index_get() -> &'static str {
+    "hello from cloudevents server"
```

Comment:
Can you make this return a `Result<Event>` instead, please? See my other comment. Thanks!